src/Commands/ImportBrands.php

```diff
@@ -2,34 +2,53 @@
 
 namespace Rapidez\Statamic\Commands;
 
+use Illuminate\Support\Facades\DB;
 use Statamic\Facades\Site;
 use Illuminate\Console\Command;
 use Rapidez\Statamic\Models\Brand;
 use TorMorten\Eventy\Facades\Eventy;
+use Illuminate\Support\Facades\Storage;
+use Statamic\Facades\Asset;
 use Rapidez\Statamic\Actions\StatamicEntryAction;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Schema;
 
 class ImportBrands extends Command
 {
     protected $signature = 'rapidez:statamic:import:brands {--site=* : Sites handles or urls to process, if none are passed it will be done for all sites (Default will be all sites)}';
 
     protected $description = 'Create a new brand entry for a brand if it does not exist.';
-    
+
     public function handle(StatamicEntryAction $statamicEntryAction): int
     {
         $sites = $this->option('site');
 
         $sites = $sites
             ? array_filter(array_map(fn($handle) => (Site::get($handle) ?: Site::findByUrl($handle)) ?: $this->output->warning(__('No site found with handle or url: :handle', ['handle' => $handle])), $sites))
             : Site::all();
-        
+
         $bar = $this->output->createProgressBar(count($sites));
 
         $bar->start();
-        
+
         foreach ($sites as $site) {
             $bar->display();
-            
+
             foreach (Brand::get() as $brand) {
+                $extraData = [];
+
+                if (Schema::hasTable('amasty_amshopby_option_setting')) {
+                    $amastyData = DB::table('amasty_amshopby_option_setting')->where('value', '=', $brand->option_id)->first();
+                    $extraData['meta_title'] = $amastyData->meta_title;
+                    $extraData['meta_description'] = $amastyData->meta_description;
+
+                    if ($amastyData->image) {
+                        $file = file_get_contents(config('rapidez.media_url').'/amasty/shopby/option_images/'.$amastyData->image, false, stream_context_create(['ssl' => ['verify_peer' => false, 'verify_peer_name' => false]]));
```

Comment:
This will break when the image cannot be downloaded for any reason, like it being missing, or the host not reachable.

Is this intended? if not, take a look at how we download the image in the image resizer module https://github.com/rapidez/image-resizer/blob/023cf6a6d4111ab523d3f5659cf2343e2ff72301/src/Controllers/ImageController.php#L131